react/features/overlay/components/OverlayContainer.js

```diff
@@ -15,8 +15,44 @@ import UserMediaPermissionsFilmstripOnlyOverlay
 import UserMediaPermissionsOverlay from './UserMediaPermissionsOverlay';
 
 /**
- * Implements a React Component that will display the correct overlay when
- * needed.
+ * Reference to the lazily loaded list of overlays.
+ */
+let _overlays;
```

Comment:
This is not an approach I'd like to encourage because it's going to be detrimental to our mobile efforts. Please don't use the approach. I discourage global values that are meant to be "runtime computed" like here `_overlays` depends on `interfaceConfig.filmStripOnly`. The only ones I encourage are global values that are meant to be "compile-time computed".

Reply:
Damn, my bad. You are absolutely right.